XCUITests/TopTabsTest.swift

```diff
@@ -63,7 +63,7 @@ class TopTabsTest: BaseTestCase {
         waitforExistence(app.webViews.links.staticTexts["More information..."])
         app.webViews.links.staticTexts["More information..."].press(forDuration: 5)
         app.buttons["Open in New Tab"].tap()
-        app.buttons["Switch"].tap()
+        sleep(2)
```

Comment:
if waitForExistence doesn't work, try removing that line altogether (i.e., no tapping 'Switch', no waits) technically it should still work